### ISSUE TITLE
Manage Collecting Event tool is not working

### ISSUE THREAD

**Issue by acdoll** [ArctosDB/arctos#2251]
On the Specimen Results page, in the "Manage..." dropdown we have the option to edit Collecting Events for the selected specimens. This tool is not working. I believe it is due to the change in our names for Specimen/Event Type that was made a while ago (e.g. from 'accepted place of collection' to just 'collection'). @dustymc , does that make sense and can we get that patched? Thanks

**Comment by dustymc** [comment]
I need more information - I'm not immediately seeing any reason it shouldn't work, but without a test box...

Is there something I can try to change with the tool?

**Comment by sharpphyl** [comment]
Andy, this morning I tried to recreate the problem that we had last week, but everything worked perfectly!  I'll keep at this and see what triggered my problems last week, but as of today it's working.  If I don't come across the same problem in a few days, we can close this.  

Dusty, when you consolidate the collecting events, does Arctos automatically delete the ones that are now unused?

**Comment by dustymc** [comment]
Yes - http://handbook.arctosdb.org/documentation/collecting-event.html#maintenance
